src/useTimer.ts

```diff
@@ -16,43 +17,38 @@ export interface IValues {
   time: number;
 }
 
-const initialConfig = {
-  endTime: null,
+const initialConfig: IConfig = {
+  endTime: 0,
```

Comment:
endTime must stay initialized to null. Timer can be incremental or decremental, so there is no default end time.